routes/tasks.ts

```diff
@@ -38,6 +38,15 @@ router.post('/', async (_req: Request, res: Response, next: NextFunction): Promi
     }
 });
 
+router.put('/update/:taskId', async (_req: Request, res: Response, next: NextFunction): Promise<void> => {
+    try {
+        const task = await Task.findByIdAndUpdate({_id: _req.params.taskId}, _req.body);
+        res.status(200).send(task);
```

Comment:
have you double checked that ti sends back the updated task or sends back the old one but after page refresh it updates?